### ISSUE TITLE
Accessibility issues with NuGet PM UI

### ISSUE THREAD

**Issue by anangaur** [NuGet/Home#7853]
Issues reported from Program-L
- [x] The tabs at the top should have more standard keyboard controls; left and right arrow moves focus to the next/previous tab when your focus is on the control, instead of each tab being individually focusable with the tab key. - fixed in 5.5 / vs16.5 by @donnie-msft's addition of the real tab control.
- [ ] I was surprised to find that ctrl+tab moves focus to the next document in your editor, instead of to the next page of the multitab dialog. Unfortunately when pressing ctrl+tab to cycle back, NVDA can't track the keyboard focus anymore; pressing the tab key does visually move focus around the dialog, but NVDA remains silent.
_**Filed a bug against NVDA reporting this behavior nvaccess/nvda#11109**_
- [ ] When arrowing through the list of packages, NVDA only announces the change in focus a few seconds after focus is visually moved, so navigating this list is painful. Again, the progress of searches being announced is a nice touch, but it feels really sluggish when moving about.
_**Filed a bug against NVDA reporting this behavior nvaccess/nvda#11109**_
- [x] NVDA only announces the package's name when arrowing through the list, but there is other information visually displayed that is not read - fixed in 5.5/vs16.5 by @rrelyea's changes in PackageItemControl.xaml
- [ ] The grid columns for selecting projects to install the package to are keyboard focusable when pressing shift tab from the list when they shouldn't be.
_**Related? https://github.com/NuGet/Home/issues/10202**_
- [x] The status of the checkbox for selecting projects to install the package to isn't indicated by NVDA. - @rrelyea added an automationproperties.name to that checkbox in 5.5/vs16.5
- [ ] The tab and shift+tab keys should not be moving focus through the list of projects. _**Related https://github.com/NuGet/Home/issues/10202**_
- [x] There is a silent tab stop between the list of projects, and the link for the package's license. - fixed in 5.5/vs16.5
- [x] The description of the package is not read at all by NVDA. https://github.com/NuGet/Home/issues/8517

**Comment by nkolev92** [comment]
I just created a new issue for the last bullet point: https://github.com/NuGet/Home/issues/8517

**Comment by nkolev92** [comment]
I think the ADO issues assigned to @zkat are duplicates of this one. 
@rrelyea I think it's a good idea to consolidate them.

**Comment by zkat** [comment]
Grabbing this one for now so I don't forget! Thanks for the ping.

**Comment by donnie-msft** [comment]
> * NVDA only announces the package's name when arrowing through the list, but there is other information visually displayed that is not read.

Does NVDA read the Installed version and/or the Latest version so that you know which is which?
May be related: https://github.com/NuGet/Home/issues/8538

**Comment by nkolev92** [comment]
reads the installed one, an issue nonetheless :)

**Comment by donnie-msft** [comment]
@anangaur for the 4 remaining tasks, I've added issues I believe are relevant. Do you happen to know the source of the tasks you listed here? Some of them are contradictory to recent reports.

**Comment by donnie-msft** [comment]
Checked with anangaur and the context on these issues is lost. I've linked issues I think are relevant. 
Closing as duplicate
